PolyEngine/Engine/Src/Engine.hpp

```diff
@@ -123,6 +123,11 @@ namespace Poly
 		/// <param name="pos">New wheel position.</param>
 		void UpdateWheelPos(const Vector& pos) { InputEventsQueue.PushBack({eInputEventType::WHEELMOVE, pos}); }
 
+		//functions for closing the game
+		bool ShouldQuit();
+		void QuitGame();
```

Comment:
Let's rename it to `RequestGameQuit`. Also please provide XML documentation (simmilar to other methods in this file).

Reply:
Hm, I don't know... `RequestGameQuit()`/`IsGameQuitRequested()` is grammatically incorrect and sounds like quitting is optional and can be denied.
How about `QuitGame()`/`GameAboutToQuit()` or `QuitGame()`/`IsGameQuitting()`?

Reply:
Apparently _quit_ as a noun is a collective term for some types of bird species, as I was surprised to find out 😛